src/structures/APIMessage.js

```diff
@@ -108,25 +108,27 @@ class APIMessage {
     const isCode = typeof this.options.code !== 'undefined' && this.options.code !== false;
     const splitOptions = isSplit ? { ...this.options.split } : undefined;
 
-    let mentionPart = '';
+    let replyPrefix = '';
     if (this.options.reply && !this.isUser && this.target.type !== 'dm') {
       const id = this.target.client.users.resolveID(this.options.reply);
-      mentionPart = `<@${this.options.reply instanceof GuildMember && this.options.reply.nickname ? '!' : ''}${id}>, `;
-      if (isSplit) {
-        splitOptions.prepend = `${mentionPart}${splitOptions.prepend || ''}`;
-      }
+      replyPrefix = this.options.replyPrefixer
+        ? this.options.replyPrefixer(this.options.reply)
+        : this.target.client.options.replyPrefixer
+        ? this.target.client.options.replyPrefixer(this.options.reply)
+        : `<@${this.options.reply instanceof GuildMember && this.options.reply.nickname ? '!' : ''}${id}>, `;
```

Comment:
```suggestion
      const prefixer = this.options.replyPrefixer || this.target.client.options.replyPrefixer;
      replyPrefix = prefixer
        ? prefixer(this.options.reply)
        :  `<@${this.options.reply instanceof GuildMember && this.options.reply.nickname ? '!' : ''}${id}>, `;
```
a bit cleaner, easier to follow